docs/metadata.md

```diff
@@ -392,22 +400,40 @@ schema = metadata.MetadataSchema(
                 "binaryFormat": "10p",
                 "pattern": "^([1-9][0-9]{3})-(1[0-2]|0[1-9])-(3[01]|0[1-9]|[12][0-9])?$",
             },
+            "location": {
+                "description": "latitude and longitude of the collected sample",
```

Comment:
Not sure this is a great example - we want people to use Individual.location for things like this, right?

Reply:
That's a good point. Have you got a suggestion for a parameter that uses one (or a set of) floating point values that we could use?

Reply:
Some kind of phenotype maybe. Height and bmi or something?

Reply:
Perhaps height and sex (so that we can use an enum)?

Reply:
Whatever - but lets not choose something divisive as an example.

Reply:
Good point. I forget this is an arguable point.